app/riot/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@storybook/riot",
+  "version": "4.0.0-alpha.18",
+  "description": "Storybook for riot.js: View riot snippets in isolation with Hot Reloading.",
+  "homepage": "https://github.com/storybooks/storybook/tree/master/app/riot",
+  "bugs": {
+    "url": "https://github.com/storybooks/storybook/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storybooks/storybook.git"
+  },
+  "license": "MIT",
+  "main": "dist/client/index.js",
+  "bin": {
+    "build-storybook": "./bin/build.js",
+    "start-storybook": "./bin/index.js",
+    "storybook-server": "./bin/index.js"
+  },
+  "scripts": {
+    "prepare": "node ../../scripts/prepare.js"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0-rc.2",
+    "@storybook/core": "4.0.0-alpha.18",
+    "common-tags": "^1.8.0",
+    "global": "^4.3.2",
+    "moment": "^2.22.2",
+    "riot": "^3.11.1",
```

Comment:
I think a few of these `riot` deps should be peed deps.

Reply:
maybe

Reply:
done I think